.gitignore

```diff
@@ -14,7 +14,7 @@ __pycache__/
 
 # data files
 numpyro/examples/.data
```

Comment:
nit: move this .data folder too

Reply:
@fehiepsi - Note that all example utilities like `datasets.py` will still  be in `numpyro/examples` and the location of the `.data` will therefore not change. This is to make it easy for all examples to reuse these utilities (the `examples` folder isn't a python module), and also to share datasets.

Reply:
Ah, got it! Thank you for clarifying!

Reply:
Ah, got it! Thank you for clarifying!